redCOMPONENT-COM/redSHOP
# Fixing product types 7

How to test:
0. be sure that reddesing product type plugin is installed,
1. save in a same category regular redSHOP products and products with redDESIGN product type selected and redDESIGN design type selected,
2. go to frontend category view and check if click on the redSHOP product with redDESIGN product type will redirect to redDESIGN frontend editor for the correct design type.
